### ISSUE TITLE
[Preference] Don't put background tab titles on a timer.

### ISSUE THREAD

**Issue by trishume** [Discordius/Lesswrong2#82]
I notice that when a LessWrong tab isn't focused, the title alternates on a timer between the article title and "LessWrong". 

This makes me sad for a number of reasons:
- The favicon already tells me the site, I want to distinguish tabs by title all the time. I acknowledge that Safari doesn't show favicons, but they use a weird title algorithm that can pick out the site name from the middle of the title if you put it in the right place.
- The flashing is distracting. This isn't too bad for me, although I can imagine it being so if I queued up a large number of tabs (they're out of phase). However, I remember at one point Scott complained about the intermittent flashing "saving..." in Google forms made it unbearably distracting for him to use. I imagine the flashing titles, especially multiple of them, might have a similar effect for a lot of people.
- I open a lot of tabs as a kind of "to read" list. I'd like it if those tabs took up only memory and no battery life, however minimal the time and energy to update the title (probably mostly graphics path rendering in the browser).

 I acknowledge that in Safari the flashing tab design allows you to see both the title and the website easier, but I think the downsides outweigh the benefits. I'm not aware of any benefits in other browsers but I could be missing something. I think it would be better if the title was always something like `Article Title - LessWrong`.

As an alternative compromise solution, consider only enabling flashing titles for Safari users.

**Comment by Discordius** [comment]
Hmm, I am not getting the same behavior. Do you have a screenshot or a specific page on which this happened?

**Comment by trishume** [comment]
![lesswrong_issue82](https://user-images.githubusercontent.com/887610/30499914-4f2799c4-9a2a-11e7-901f-c49ae19814d1.gif)

In the course of making a demo gif, I figured out the reason. When I dragged the tabs to another window (that doesn't have a ton of tabs already), the larger tabs allowed me to see that the flashing title was "LessWrong2.0 says ...", which suggests that the problem is Intercom.

It appears the solution is either to stop intercom from doing this, or to not send a fake intro message on Intercom whenever you visit the site. I prefer not having the fake message, it's annoying. I'm okay with just the intercom button, and maybe a less intrusive message you don't have to dismiss somewhere else.
